app/views/votes/_voting.html.erb

```diff
@@ -1,11 +1,10 @@
 <% if @user_votes.include? vote %>
-  <% Vote::VOTE_TYPES.each do |reason, value| %>
+  <% t(:votes).each do |value, reason| %>
     <%= link_to reason, quote_vote_path(quote, vote, { vote: { value: value } } ), method: :patch %>
   <% end %>
   <%= link_to "Erase my vote", quote_vote_path(quote, vote), method: :delete %>
 <% else %>
-  <% Vote::VOTE_TYPES.each do |reason, value| %>
+  <% t(:votes).each do |value, reason| %>
```

Comment:
I don't know if it pings you just for me referencing this PR somewhere else but I did this a different, possibly better way in #32 .  Just didn't push over this since I didn't want to totally erase the I18n stuff and it was still possibly being discussed?  I do like the new method implemented though.